### PR TITLE
Modpacks Info

### PR DESCRIPTION
**Pull request by Ladvace**
downloads, author....

### CODE REVIEW

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Clones added
============
- src/common/modals/ModOverview.js  3
- src/common/modals/ModpackDescription.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474&bid=19215118)

File: src/common/modals/ModpackDescription.js [hunk 1]
```diff
@@ -11,6 +14,25 @@ import CloseButton from '../components/CloseButton';
 import { closeModal } from '../reducers/modals/actions';
 import { FORGE, CURSEFORGE_URL } from '../utils/constants';
 
+const formatNumber = number => {
+  // Alter numbers larger than 1k
+  if (number >= 1e3) {
```
Comment (killpowa) [inline src/common/modals/ModpackDescription.js:19]: Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/modals/ModOverview.js [hunk 2]
```diff
@@ -73,6 +78,27 @@ const ModOverview = ({
     }
   };
 
+  const formatNumber = number => {
+    // Alter numbers larger than 1k
+    console.log('pino', number);
+    if (number >= 1e3) {
```
Comment (killpowa) [inline src/common/modals/ModOverview.js:84]: Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Clones added
============
- src/common/modals/ModOverview.js  3
- src/common/modals/ModpackDescription.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474&bid=19215118)

File: src/common/modals/ModpackDescription.js [hunk 3]
```diff
@@ -11,6 +14,36 @@ import CloseButton from '../components/CloseButton';
...
```
Comment (killpowa) [inline src/common/modals/ModpackDescription.js:19]: Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/modals/ModOverview.js [hunk 4]
```diff
@@ -73,6 +78,38 @@ const ModOverview = ({
...
```
Comment (killpowa) [inline src/common/modals/ModOverview.js:84]: Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           

Clones added
============
- src/common/modals/ModOverview.js  3
- src/common/modals/ModpackDescription.js  3
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474&bid=19215118)

File: src/common/modals/ModpackDescription.js [hunk 5]
```diff
@@ -11,6 +14,35 @@ import CloseButton from '../components/CloseButton';
...
```
Comment (killpowa) [inline src/common/modals/ModpackDescription.js:19]: Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/modals/ModOverview.js [hunk 6]
```diff
@@ -73,6 +78,35 @@ const ModOverview = ({
     }
   };
 
+  const formatNumber = number => {
+    // Alter numbers larger than 1k
+    if (number >= 1e3) {
```
Comment (killpowa) [inline src/common/modals/ModOverview.js:83]: Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

**Comment by killpowa** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity increasing per file
==============================
- src/common/modals/ModsChangelogs.js  2
         

Clones added
============
- src/common/modals/ModpackDescription.js  2
- src/common/modals/ModOverview.js  2
- src/common/utils/index.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474&bid=19215118)

File: src/common/modals/ModOverview.js [hunk 7]
```diff
@@ -38,14 +47,35 @@ const ModOverview = ({
   const instancesPath = useSelector(_getInstancesPath);
   const instance = useSelector(state => _getInstance(state)(instanceName));
 
+  const initScreenShots = async data => {
+    if (data) {
+      const mappedFiles = await Promise.all(
+        data.map(async v => {
+          const { data: changelog } = await getAddonFileChangelog(
```
Comment (killpowa) [inline src/common/modals/ModOverview.js:54]: Codacy found an issue: [Unnecessary block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/modals/ModOverview.js [hunk 8]
```diff
@@ -38,14 +47,35 @@ const ModOverview = ({
...
+            projectID,
+            v.id
+          );
+          return {
```
Comment (killpowa) [inline src/common/modals/ModOverview.js:58]: Codacy found an issue: [Unnecessary block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/modals/ModpackDescription.js [hunk 9]
```diff
@@ -3,31 +3,62 @@ import React, { useState, useEffect } from 'react';
...
   const [files, setFiles] = useState(null);
+  const [changeLog, setChangeLog] = useState(null);
   const [selectedId, setSelectedId] = useState(false);
   const [loading, setLoading] = useState(false);
 
+  const initScreenShots = async data => {
+    if (data) {
+      const mappedFiles = await Promise.all(
+        data.map(async v => {
+          const { data: changelog } = await getAddonFileChangelog(
```
Comment (killpowa) [inline src/common/modals/ModpackDescription.js:34]: Codacy found an issue: [Unnecessary block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/modals/ModpackDescription.js [hunk 10]
```diff
@@ -3,31 +3,62 @@ import React, { useState, useEffect } from 'react';
...
+            modpack.id,
+            v.id
+          );
+          return {
```
Comment (killpowa) [inline src/common/modals/ModpackDescription.js:38]: Codacy found an issue: [Unnecessary block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/utils/index.js [hunk 11]
```diff
@@ -24,6 +24,33 @@ export function sortByForgeVersionDesc(a, b) {
   return 0;
 }
 
+export const formatNumber = number => {
+  // Alter numbers larger than 1k
+  if (number >= 1e3) {
```
Comment (killpowa) [inline src/common/utils/index.js:29]: Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

**Comment by blarfoon** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 5
           

Complexity increasing per file
==============================
- src/common/modals/ModsChangelogs.js  2
         

Clones added
============
- src/common/utils/index.js  2
- src/common/modals/ModOverview.js  2
- src/common/modals/ModpackDescription.js  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474&bid=19215118)

File: src/common/modals/ModOverview.js (hunk 7 above)
Comment (blarfoon) [inline src/common/modals/ModOverview.js:54]: Codacy found an issue: [Unnecessary block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/utils/index.js (hunk 11 above)
Comment (blarfoon) [inline src/common/utils/index.js:29]: Codacy found an issue: [The numeric literal '1e3' will have at different value at runtime.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/modals/ModpackDescription.js (hunk 10 above)
Comment (blarfoon) [inline src/common/modals/ModpackDescription.js:38]: Codacy found an issue: [Unnecessary block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/modals/ModpackDescription.js (hunk 9 above)
Comment (blarfoon) [inline src/common/modals/ModpackDescription.js:34]: Codacy found an issue: [Unnecessary block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

File: src/common/modals/ModOverview.js (hunk 8 above)
Comment (blarfoon) [inline src/common/modals/ModOverview.js:58]: Codacy found an issue: [Unnecessary block.](https://app.codacy.com/gh/gorilla-devs/GDLauncher/pullRequest?prid=5870474)

**Comment by furtabs** [comment]
:+1: Its another decently good PR with framework for Modpacks.ch and Technic Support. Too Bad Davide isnt here to merge it

Edit: Just capitalise MC and you have my approval

**Pull request merged**